### PR TITLE
Fix claimable reward issue

### PR DESCRIPTION
**Pull request by Emmanueldmlr**
- [x] The claimable rewards going to zero on the claim 

- [x] Gas on the header is not showing anything 

fixes https://github.com/EveripediaNetwork/issues/issues/908

### CODE REVIEW

**Comment by Emmanueldmlr** [comment]
For Gas
image

**Comment by Emmanueldmlr** [comment]
Uploading Screen Recording 2022-11-30 at 5.40.30 PM.mov…

**Comment by Emmanueldmlr** [comment]
https://user-images.githubusercontent.com/70170061/204857695-4e040753-07f0-4eae-85db-2d7503fffe8d.mov

File: src/pages/api/gas-price.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  const url = `https://pro-openapi.debank.com/v1/wallet/gas_market?chain_id=eth`
+  const result = await fetch(url, {
+    headers: {
+      Accept: 'application/json',
+      Accesskey: `${config.debankApiKey}`,
+    },
+  })
+  const response = await result.json()
+  const gasPrice = response[1].price / NORMALIZE_VALUE
+  res.setHeader('Cache-Control', 's-maxage=43200')
```
Comment (kesar) [inline src/pages/api/gas-price.ts:31]: in this case I would suggest a cache of max 1 min.

**Pull request merged**